web/src/routes/Event.jsx

```diff
@@ -8,19 +13,76 @@ import { Table, Thead, Tbody, Th, Tr, Td } from '../components/Table';
 export default function Event({ eventId }) {
   const apiHost = useApiHost();
   const { data, status } = useEvent(eventId);
+  const [showDialog, setShowDialog] = useState(false);
+  const [deleteStatus, setDeleteStatus] = useState(FetchStatus.NONE);
+  const [deleteMessage, setDeleteMessage] = useState();
```

Comment:
You're calling `setDeleteMessage()`, but the value is never used. Should be able to safely remove this